ui/src/app/utils/swagger-date.ts

```diff
@@ -0,0 +1,33 @@
+
+// Inferred structure of JSON object coming down the wire for a date-time in Swagger. It's apparently
+// not a Date as advertised.
+export interface DateObjectProps {
```

Comment:
If this is actually what we're getting on the wire, then surely it's a bug with the JSON serialization on the server; I can't imagine this verbose payload is the expect wire format for Swagger. This seems like a serialization of some internal Java object representation, i.e. the JSON serialization (presumably for java8 dateLibrary) with date-time does not work.

If it's a serialization issue, then patching over it on the client side seems incorrect to me.

For now, it may be easier for consistency to follow the unix millis pattern used in the rest of the API (though I would prefer we eventually switch to a string representation instead per https://precisionmedicineinitiative.atlassian.net/browse/RW-4682):
```
      creationTime:
        type: integer
        format: int64
        description: Milliseconds since the UNIX epoch.
```

Reply:
I agree serialization is the  place to look. i had the perspective that like the Hitchhiker's Guide, if the JSON is wrong, it's definitively wrong.

Yes this is a band aid, but I feel like making breaking changes to the API for a hotfix is worse. The blast radius of the fix will be significantly higher.

For the purposes of RW-4682, I've removed Joda already from everything except Cohort Builder, and `java.time.OffsetDateTime` is now the standard on the backend.

Note that this serialization was broken (in a different way) before the server-side change, where it was somehow sending a millisecond value instead of the verbose format.

In  any case, I'd  prefer to start with this fix, because the audit trail is very difficult to use today if you'er not familiar with Chrome dev tools, and I don't really have more time allocated this sprint for it.

Reply:
How would the blast radius be significantly higher? The server-client communication is already broken and this is the only page that uses the value.

Reply:
Reverting the change to the Java8 is the main thing I want to avoid, since the reporting stuff uses it in  several places.

Reply:
My suggestion was to switch the type to int for consistency with our other timestamps.

Reply:
Yeah, I can do that for now.